### ISSUE TITLE
Remove pushgateway

### ISSUE THREAD

**Issue by yeya24** [pingcap/tidb-docker-compose#83]
I think push gateway component is not necessary now since TIKV supports Prometheus `pull` mode now.

**Comment by Anutrix** [comment]
Yes. Also want to add that the version we use(v0.3.1) is 5 years old.
